pkg/controllers/management/secretmigrator/authconfig_test.go

```diff
@@ -0,0 +1,382 @@
+package secretmigrator
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/rancher/norman/objectclient"
+	apimgmtv3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/rancher/pkg/auth/providers/saml"
+	corefakes "github.com/rancher/rancher/pkg/generated/norman/core/v1/fakes"
+	"github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3/fakes"
+	"github.com/rancher/rancher/pkg/namespace"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+const (
+	mockPass                = "testpass1234"
+	testCreationStampString = "2023-05-15T19:28:22Z"
+)
+
+func TestSetUnstructuredStatus(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name   string
+		input  unstructuredConfig
+		output unstructuredConfig
+	}{
+		{
```

Comment:
Would be good to have the other tests cases too (missing status for example), not sure if those were lost in the merge

Reply:
Ah...yes....thanks.  I accidentally whacked them.